src/Slices/EditInstance/UI/EditForm.tsx

```diff
@@ -25,7 +25,7 @@ export const EditForm: React.FC<Props> = ({ serviceEntity, instance }) => {
   const { commandResolver, environmentModifier, routeManager } =
     useContext(DependencyContext);
 
-  const fieldCreator = new FieldCreator(new EditModifierHandler());
+  const fieldCreator = new FieldCreator(new EditModifierHandler(), true);
```

Comment:
what does the true stand for ? In this case, it would make sense to have a named argument.

Reply:
that boolean check if ```isDisabled``` logic should be taken into account, as FieldCreator is for both, ```EditForm``` and ```CreateInstance```. Named arguments collides with ```private readonly``` parameters for constructor, so I can either remove those params, or add comment above initialisation here

Reply:
yeah, sure, a comment is fine for me. 
I thought you could do : 
```suggestion
  const isDisabled = true;
  const fieldCreator = new FieldCreator(new EditModifierHandler(), isDisabled);
```

Reply:
Oh, I thought you meant to change the props to fieldCreator to named arguments, I'll do that then :D